bin/assigner/src/main.cpp

```diff
@@ -661,20 +668,34 @@ int main(int argc, char *argv[]) {
         public_input_file_name = vm["public-input"].as<std::string>();
     }
 
-    if (vm.count("assignment-table")) {
-        assignment_table_file_name = vm["assignment-table"].as<std::string>();
-    } else {
-        std::cerr << "Invalid command line argument - assignment table file name is not specified" << std::endl;
-        std::cout << options_desc << std::endl;
-        return 1;
-    }
-
-    if (vm.count("circuit")) {
-        circuit_file_name = vm["circuit"].as<std::string>();
+    if (vm.count("estimate_size")) {
+        if (vm.count("assignment-table")) {
+            std::cerr << "Assigner is running with --estimate_size flag and will not generate .tbl assignment table.\n";
```

Comment:
I guess better allow user set output files (*.tbl, *.crct) even set `--estimate-size` flag. It's not error, may be just warning.
```
if (vm.count("assignment-table")) {
        assignment_table_file_name = vm["assignment-table"].as<std::string>();
    } else if (!vm.count("estimate_size")) {
        std::cerr << "Invalid command line argument - assignment table file name is not specified" << std::endl;
        std::cout << options_desc << std::endl;
        return 1;
    }
```